Scripts/build-phases/swiftlint.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+#
+# Runs SwiftLint on the whole workspace
```

Comment:
Nitpick, feel free to ignore. The comment below has a `.` 😳

```suggestion
# Runs SwiftLint on the whole workspace.
```

Reply:
Thanks! Fixed in 1896de78f06f9bf290bbe4e3d3ef6b37971d3d22.